project_event/wizard/project_event_wizard.py

```diff
@@ -20,6 +21,10 @@ class ProjectEventWizard(models.TransientModel):
         'res.partner',
         string='Responsible',
     )
+    event_partner_id = fields.Many2one(
+        'res.partner',
+        string='Client',
+    )
     event_notes = fields.Text(
         string='Notes',
     )
```

Comment:
In the method add_task():

```
 for activity in activities:
    map(lambda self, task: env['project.task.wizard'].create({
'event_wizard_id': self.id,
                     'template_id': task.id,	                     'template_id': task.id,
                     'activity_wiz_id': act.id,	                     'activity_wiz_id': act.id,
                     'task_name': task.name,	                     'task_name': task.name,
                     'task_resp_id': task.temp_resp_id.id,	                     'task_resp_id': task.temp_resp_id.id,
                     'task_category_id': task.task_category_id.id,	                     'task_category_id': task.task_category_id.id,
                     'resource_type': task.resource_type,	                     'resource_type': task.resource_type,
                     'equipment_id': task.equipment_id.id,	                     'equipment_id': task.equipment_id.id,
                     'room_id': task.room_id.id,	                     'room_id': task.room_id.id,
                     'department_id': task.department_id.id,	                     'department_id': task.department_id.id,
                     'duration': task.duration,	                     'duration': task.duration,
                     'start_time': task.start_time,	                     'start_time': task.start_time,
                     'notes': task.notes,	                     'notes': task.notes,
}), activity.template_id.task_template_ids)

```



the nested for loop could be better written like `this:`
map is useful when you want to apply the function to every item of an iterable and return a list of the results. This is simpler and more concise than using a for loop and constructing a list.